lib/iris/tests/test_pp_cf.py

```diff
@@ -81,7 +81,7 @@ def callback_aaxzc_n10r13xy_b_pp(cube, field, filename):
 
 
 # XXX: Issue with integer.b.pp and invalid bmdi fill_value for int32 dtype
-@tests.skip_biggus
+#@tests.skip_biggus
```

Comment:
Change got in by mistake (hence test fail). 
Now removed this + pushed modified version...